### ISSUE TITLE
Falsy attributes are rendered by pelo

### ISSUE THREAD

**Issue by bcomnes** [shuhei/pelo#8]
Reposted here from https://github.com/choojs/choo/issues/542 by @tornqvist

Since not actually parsing the html is what makes pelo fast compared to bel, I'm not quite sure how this could be addressed but nonetheless it's an issue that output differs. Also these kinds of patterns, using booleans to determine i.e. input disabled state is quite common, at least in my case.

### Expected behavior

Elements with interpolated falsy (`null`/`false`) values be excluded from output.

 ```javascript
html`<input disabled=${ false } />`
```

### Actual behavior

```html
<!-- pelo output -->
<input disabled="" />

<!-- bel output -->
<input />
```

Pelo renders attributes with falsy values as `attribute=""` whereas bel completely excludes them.

### Steps to reproduce behavior
See requirebin: http://requirebin.com/?gist=1dc8acd84ec5bb811458131431bffc4b

**Comment by shuhei** [comment]
Thanks for letting me know! Mmm, this is a very fundamental issue for `pelo`... If `pelo` starts parsing HTML, then it will lose its point.

One idea is to remove `disabled=""`, `checked=""`, etc. by string replace.

**Comment by shuhei** [comment]
Fixing with b5747acff76449fb10b3fcf635e76f403bb5d9b9

**Comment by shuhei** [comment]
Hopefully it's fixed in `pelo@0.0.3`.

**Comment by bcomnes** [comment]
Whoa cool Thanks for the quick fix.
